doc/source/ray-core/tasks/generators.rst

```diff
@@ -61,15 +61,15 @@ We may not know the size of the argument until we execute the task, so we don't
 
 In these cases, we can use a remote generator function that returns a *dynamic* number of values.
 To use this feature, set ``num_returns="dynamic"`` in the ``@ray.remote`` decorator or the remote function's ``.options()``.
-Then, when invoking the remote function, Ray will return a *single* ``ObjectRef`` that will get populated with an ``ObjectRefGenerator`` when the task completes.
-The ``ObjectRefGenerator`` can be used to iterate over a list of ``ObjectRefs`` containing the actual values returned by the task.
+Then, when invoking the remote function, Ray will return a *single* ``ObjectRef`` that will get populated with an ``DynamicObjectRefGenerator`` when the task completes.
```

Comment:
```suggestion
When you invoke the remote function, Ray returns a *single* ``ObjectRef`` that is populated with a ``DynamicObjectRefGenerator`` when the task completes.
```

Reply:
this file is not relevant anymore. wont fix